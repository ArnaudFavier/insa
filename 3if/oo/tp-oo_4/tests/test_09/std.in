S segment -3 1 5 1
R rectangle -8 4 4 -4
PC poly 1 1 -1 2 -3 0 -1 -3 1 0
OR union segment rectangle poly
OI inter segment rectangle poly
LIST
DELETE nothing
DELETE inter
LIST
DELETE segment
LIST
CLEAR
LIST
EXIT
