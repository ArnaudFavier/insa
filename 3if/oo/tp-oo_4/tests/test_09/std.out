OK
OK
OK
OK
OK
inter I [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
ERR
OK
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
segment S [(-3;1) (5;1)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
poly C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]
rectangle R [(-8;4) (4;-4)]
union U [S [(-3;1) (5;1)] R [(-8;4) (4;-4)] C [(1;1) (-1;2) (-3;0) (-1;-3) (1;0)]]
OK
